gui/wxpython/main_window/frame.py

```diff
@@ -614,7 +614,7 @@ def BuildPanes(self):
             self.datacatalog,
             aui.AuiPaneInfo()
             .Name("datacatalog")
-            .Caption("Data Catalog")
+            .Caption("Data")
```

Comment:
I think these should be translatable. At least they are translatable in multiple-window GUI.

Reply:
Thanks for catching that. None of the pane captions were translatable. Fixed.